package.json

```
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "npx gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-status-codes 0,403,429,503,302 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files \"//select-a-language//,//2017//,//2018//,//2019//,//code-guidelines//\" ./_site",
+    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,403,429 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/2017/,/2018/,/code-guidelines/' ./_site",
```

When I remove `/code-guidelines` from the ignore list and run `npm run proof` locally, I get a single failure with status code 200. [Status code 200](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/200) means "successful", and I wonder if it is safe to remove `code-guidelines` from the ignore list and instead add status 200.

``` 
* At ./_site/documentation/code-guidelines/index.html:2815:

  External link https://github.com/uswds/uswds/blob/develop/LICENSE.md#the-rest-of-this-project-is-in-the-worldwide-public-domain failed: https://github.com/uswds/uswds/blob/develop/LICENSE.md exists, but the hash 'the-rest-of-this-project-is-in-the-worldwide-public-domain' does not (status code 200)
```

```suggestion
    "proof": "bundle exec htmlproofer --enforce-https=false --allow-missing-href=true --ignore-status-codes 0,200,403,429 --swap-urls 'https\\://designsystem.digital.gov/:/' --ignore-urls '/github.com/uswds/uswds/issues/' --ignore-files '/2017/,/2018/' ./_site",
```